aws-crt-kotlin/build.gradle.kts

```diff
@@ -24,42 +23,6 @@ configureKmpTargets()
 kotlin {
     explicitApi()
 
-    jvm {
-        attributes {
-            attribute<org.gradle.api.attributes.java.TargetJvmEnvironment>(
-                TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE,
-                objects.named(TargetJvmEnvironment.STANDARD_JVM),
-            )
-        }
-    }
-
-    // KMP doesn't support sharing source sets for multiple JVM targets OR JVM + Android targets.
-    // We can manually declare a `jvmCommon` target and wire it up. It will compile fine but Intellij does
-    // not support this and the developer experience is abysmal. Kotlin/Native suffers a similar problem and
-    // we can use the same solution. Simply, if Intellij is running (i.e. the one invoking this script) then
-    // assume we are only building for JVM. Otherwise declare the additional JVM target for Android and
-    // set the sourceSet the same for both but with different runtime dependencies.
-    // See:
-    //     * https://kotlinlang.org/docs/mpp-share-on-platforms.html#share-code-in-libraries
-    //     * https://kotlinlang.org/docs/mpp-set-up-targets.html#distinguish-several-targets-for-one-platform
-    if (!IDEA_ACTIVE) {
-
-        // NOTE: We don't actually need the Android plugin. All of the Android specifics are handled in aws-crt-java,
-        // we just need a variant with a different dependency set + some distinguishing attributes.
-        jvm("android") {
-            attributes {
-                attribute(
-                    org.jetbrains.kotlin.gradle.plugin.KotlinPlatformType.Companion.attribute,
-                    org.jetbrains.kotlin.gradle.plugin.KotlinPlatformType.androidJvm,
-                )
-                attribute(
-                    TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE,
-                    objects.named(TargetJvmEnvironment.ANDROID),
-                )
-            }
-        }
-    }
```

Comment:
Removed because of:

```
e: Kotlin Target 'jvm()' is already declared.

Declaring multiple Kotlin Targets of the same type is not supported.

Read https://kotl.in/declaring-multiple-targets for details.
```